### PR TITLE
Add setMaxWebAudioPlayers and introduce generic namespace for tasks

### PR DESCRIPTION
**Pull request by skywilliamsonl**
Summary: Implement the setting of the max web audio players in server manager. Add new namespace in gstplayer/tasks/generic like we do for webaudio.
Type: Feature
Test Plan: Unittests
Jira: RIALTO-2

### CODE REVIEW

**Review by skypetts — APPROVED** [review]

**Review by skypetts — APPROVED** [review]

File: media/server/gstplayer/include/tasks/generic/AttachSamples.h [hunk 1]
```diff
@@ -27,7 +27,7 @@
 #include <gst/gst.h>
 #include <vector>
 
-namespace firebolt::rialto::server
+namespace firebolt::rialto::server::generic
```
Comment (skywojciechowskim) [inline media/server/gstplayer/include/tasks/generic/AttachSamples.h:30]: This namespace is a little bit confusing for me. generic here is only for tasks, not whole rialto server. If you want to introduce a new namespace here, I would use rialto::server::tasks::generic/webaudio

File: serverManager/common/include/ISessionServerAppManager.h [hunk 2]
```diff
@@ -38,8 +38,10 @@ class ISessionServerAppManager
     ISessionServerAppManager &operator=(const ISessionServerAppManager &) = delete;
     ISessionServerAppManager &operator=(ISessionServerAppManager &&) = delete;
 
-    virtual bool setSessionServerState(const std::string &appId, const service::SessionServerState &newState) = 0;
-    virtual void onSessionServerStateChanged(const std::string &appId, const service::SessionServerState &newState) = 0;
+    virtual bool setSessionServerState(const std::string &appId,
+                                       const firebolt::rialto::common::SessionServerState &newState) = 0;
```
Comment (skywojciechowskim) [inline serverManager/common/include/ISessionServerAppManager.h:42]: In this pull request, you changed SessionServerState (one of rialto::servermanager public enums) to firebolt::rialto::common. Maybe it's worth to change all rialto::servermanager namespace occurences to firebolt::rialto::servermanager? Just a suggestion, what do others think?

Comment (skywilliamsonl) [reply serverManager/common/include/ISessionServerAppManager.h:42]: Yeah that makes sense, will do.

File: media/server/gstplayer/include/tasks/generic/AttachSamples.h (hunk 1 above)
Comment (skywilliamsonl) [reply media/server/gstplayer/include/tasks/generic/AttachSamples.h:30]: Yeah that looks clearer, I can make that change.

**Review by skywojciechowskim — APPROVED** [review]

**Review by skywojciechowskim — APPROVED** [review]

**Review by aczs — APPROVED** [review]

**Pull request merged**